### PR TITLE
CODENVY-2284: Add additional logs if start from snapshot fail

### PR DESCRIPTION
**Pull request by mmorhun**
### What does this PR do?
Adds additional logs when start from snapshot failed and the system fall back to start from recipe.

### What issues does this PR fix or reference?
https://github.com/codenvy/codenvy/issues/2284

#### Changelog
Additional logs added  if workspace start from snapshot fail.

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

File: wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java [hunk 1]
```diff
@@ -940,7 +940,9 @@ private Instance startInstance(boolean recover,
                         }
                     } catch (NotFoundException e) {
                         try {
-                            machineLogger.writeLine("Failed to boot machine from snapshot: snapshot not found. " +
+                            machineLogger.writeLine("Failed to boot machine " + machine.getConfig().getName() +
```
Comment (tolusha) [inline wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java]:
```
String.format("Failed to boot machine %s, reason: snapshot not found, workspace id=%s. Machine will be created from origin source.", 
           machine.getConfig().getName(),
           machine.getWorkspaceId())
```

File: wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java [hunk 2]
```diff
@@ -940,17 +940,22 @@ private Instance startInstance(boolean recover,
...
                         } catch (IOException ignore) { }
                     }
                 }
 
                 instance = machineStarter.startMachine(machineLogger, machineSource);
             } catch (SourceNotFoundException e) {
                 if (recover) {
-                    LOG.error("Image of snapshot for machine " + machine.getConfig().getName() +
-                              " not found. " + "Machine will be created from origin source.");
+                    LOG.error(format("Snapshot image for machine %s of workspace %s not found. " +
```
Comment (tolusha) [inline wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java]: Snapshot of the image for the machine of the workspace is not found

File: wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java [hunk 3]
```diff
@@ -940,17 +940,22 @@ private Instance startInstance(boolean recover,
...
                         } catch (IOException ignore) { }
                     }
                 }
 
                 instance = machineStarter.startMachine(machineLogger, machineSource);
             } catch (SourceNotFoundException e) {
                 if (recover) {
-                    LOG.error("Image of snapshot for machine " + machine.getConfig().getName() +
-                              " not found. " + "Machine will be created from origin source.");
+                    LOG.error(format("Snapshot of the image for the machine %s of the workspace %s is not found. " +
```
Comment (tolusha) [inline wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java]: The snapshot

File: wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java [hunk 4]
```diff
@@ -940,17 +940,22 @@ private Instance startInstance(boolean recover,
...
+                                     "Machine will be created from origin source.",
+                                     machine.getConfig().getName(),
```
Comment (tolusha) [inline wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/CheEnvironmentEngine.java]: The machine will be created from the origin source

**Review by tolusha — APPROVED** [review]

**Review by AndrienkoAleksandr — APPROVED** [review]

**Pull request merged**